libreoffice-core/sw/source/uibase/docvw/edtwin.cxx

```diff
@@ -623,16 +623,21 @@ void SwEditWin::UpdatePointer(const Point &rLPt, sal_uInt16 nModifier )
         }
 
         // which kind of text pointer have we to show - horz / vert - ?
-        if( PointerStyle::Text == eStyle && rSh.IsInVerticalText( &rLPt ))
-            eStyle = PointerStyle::TextVertical;
-        else if (rSh.GetViewOptions()->CanHideWhitespace() &&
-                 rSh.GetLayout()->IsBetweenPages(rLPt))
-        {
-            if (rSh.GetViewOptions()->IsHideWhitespaceMode())
-                eStyle = PointerStyle::ShowWhitespace;
-            else
-                eStyle = PointerStyle::HideWhitespace;
-        }
+
+
+        // NOTE: @teo turning off show/hide whitespace between pages
+        // https://www.notion.so/macrocom/Rendering-Issues-clicking-between-Pages-9eef9ecf67c54542ad74adefdefd7315
+
+        /* if( PointerStyle::Text == eStyle && rSh.IsInVerticalText( &rLPt )) */
+        /*     eStyle = PointerStyle::TextVertical; */
+        /* else if (rSh.GetViewOptions()->CanHideWhitespace() && */
+        /*          rSh.GetLayout()->IsBetweenPages(rLPt)) */
+        /* { */
+        /*     if (rSh.GetViewOptions()->IsHideWhitespaceMode()) */
+        /*         eStyle = PointerStyle::ShowWhitespace; */
+        /*     else */
+        /*         eStyle = PointerStyle::HideWhitespace; */
+        /* } */
```

Comment:
This seems to change the cursor style, with this enabled, does it crash?

Reply:
the crash is caused by the white space, i think the cursor change is to visually indicate the toggle at least that is what it looks like on libre office.